web-auth/src/components/Auth.svelte

```diff
@@ -6,69 +6,69 @@
   import { onMount } from "svelte";
   import { LOGIN_OPTIONS } from "../config";
   import AuthContainer from "./AuthContainer.svelte";
-  import Disclaimer from "./Disclaimer.svelte";
-  import EmailPassForm from "./EmailPassForm.svelte";
-  import SSOForm from "./SSOForm.svelte";
+  import EmailPasswordForm from "./EmailPasswordForm.svelte";
   import { getConnectionFromEmail } from "./utils";
-
-  type InternalOptions = {
-    protocol: string;
-    response_type: string;
-    prompt: string;
-    scope: string;
-    _csrf: string;
-    leeway: number;
-  };
-
-  type Config = {
-    auth0Domain: string;
-    clientID: string;
-    auth0Tenant: string;
-    authorizationServer: {
-      issuer: string;
-    };
-    callbackURL: string;
-    internalOptions: InternalOptions;
-    extraParams?: { screen_hint?: string };
-  };
+  import OrSeparator from "./OrSeparator.svelte";
+  import SSOForm from "./SSOForm.svelte";
+  import EmailSubmissionForm from "./EmailSubmissionForm.svelte";
+  import DiscordCTA from "./DiscordCTA.svelte";
+  import Disclaimer from "./Disclaimer.svelte";
+  import Spacer from "./Spacer.svelte";
+  import { LOCAL_STORAGE_KEY } from "../constants";
+  import { AuthStep, type Config } from "../types";
 
   export let configParams: string;
-  export let cloudClientIDs = "";
   export let disableForgotPassDomains = "";
   export let connectionMap = "{}";
 
-  const connectionMapObj = JSON.parse(connectionMap) as Record<
-    string,
-    string[]
-  >;
-  const cloudClientIDsArr = cloudClientIDs.split(",");
+  const connectionMapObj = JSON.parse(connectionMap);
   const disableForgotPassDomainsArr = disableForgotPassDomains.split(",");
 
-  // By default show the LogIn page
-  $: isLoginPage = true;
   $: errorText = "";
-  $: isRillCloud = false;
 
   let isSSODisabled = false;
   let isEmailDisabled = false;
```

Comment:
I don't see this `isEmailDisabled` variable ever get modified

Reply:
Good call, this was a remnant from the previous centralized implementation in Auth.svelte. Will remove it.

Reply:
also removed unused `isSSODisabled`